src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/bufferOutputPolling.ts

```diff
@@ -184,6 +190,25 @@ export function promptForMorePolling(command: string, context: IToolInvocationCo
 	return { promise: Promise.resolve(false) };
 }
 
+export function promptForMorePolling(title: string, message: string, context: IToolInvocationContext, chatService: IChatService): { promise: Promise<boolean>; part?: ChatElicitationRequestPart } {
+	return createYesNoPrompt(
+		context,
+		chatService,
+		title,
+		message
+	);
+}
+
+export function promptForYesNo(context: IToolInvocationContext, chatService: IChatService): { promise: Promise<boolean>; part?: ChatElicitationRequestPart } {
+	return createYesNoPrompt(
+		context,
+		chatService,
+		localize('poll.terminal.yes', 'Respond yes in the terminal?'),
```

Comment:
We could be more specific:

- Respond "yes" in the terminal?
- Respond "y" in the terminal?